README.md

```diff
@@ -29,6 +23,18 @@ If you prefer to do it straight from CLI here's a one-liner for you:
 ```bash
 curl -L https://github.com/redhat-developer/opencompose/releases/download/v0.1.0-alpha.0/opencompose-v0.1.0-alpha.0-d0edfd9-linux-64bit.tar.xz | tar -xJf - -C ${HOME}/bin ./opencompose
 ```
+#### From source
+You can easily build OpenCompose tool from source. All you need is [Go](https://golang.org/dl/):
+```sh
+go get -u github.com/redhat-developer/opencompose
+```
+
+Although the simple `go get` installation works for just trying it out, if you want to submit bug reports you have to build it the supported way:
```

Comment:
`Although `go get` instalation works for just trying it out` may sound a bit better